internal/test/envtest/config/snow-crds.yaml

```diff
@@ -20,7 +20,6 @@ spec:
     strategy: Webhook
     webhook:
       clientConfig:
-        caBundle: Cg==
```

Comment:
the `spec.conversion.webhook.clientConfig.caBundle` is using an invalid value Cg==(\n) for placeholder, which is not compatible with k8s > 1.22